pkg/operators/spoke/controller_test.go

```diff
@@ -272,22 +272,22 @@ func TestSyncWithNoSecret(t *testing.T) {
 	if err == nil {
 		t.Errorf("Expected error when sync")
 	}
-	nucleusAction = controller.nucleusClient.Actions()
-	if len(nucleusAction) != 4 {
-		t.Errorf("Expect 4 actions in the sync loop, actual %#v", nucleusAction)
+	klusterletAction = controller.operatorClient.Actions()
```

Comment:
seems like this is more accurately `operatorAction` in keeping with the original

Reply:
done